src/GuppyDataExplorer/TierAccessSelector/index.jsx

```diff
@@ -0,0 +1,118 @@
+      <div className='tier-access-selector'>
+        <h4 // eslint-disable-line jsx-a11y/no-noninteractive-element-interactions
+          className='tier-access-selector__dropdown'
+          onClick={this.handleToggle}
+        >
+          <span className='tier-access-selector__title'>Data Access</span>
+          <i className={`tier-access-selector__dropdown-icon g3-icon g3-icon--chevron-${this.state.toggled ? 'down' : 'up'}`} />
+        </h4>
+        {
+          !this.state.toggled && (
+            <div>
+              <div className='tier-access-selector__items'>
+                <div className='tier-access-selector__item'>
+                  <input
+                    onChange={this.handleSelectorChange}
+                    name='access-level'
+                    id='with-access'
+                    value='with-access'
+                    className='tier-access-selector__radio-input'
+                    type='radio'
+                    checked={this.state.selected === 'with-access'}
+                  />
+                  <i className='tier-access-selector__customized-radio-input' />
+                  <label htmlFor='with-access' className='tier-access-selector__label'>Data with Access</label>
+                </div>
+                <div className='tier-access-selector__item'>
+                  <input
+                    onChange={this.handleSelectorChange}
+                    name='access-level'
+                    id='without-access'
+                    value='without-access'
+                    className='tier-access-selector__radio-input'
+                    type='radio'
+                    checked={this.state.selected === 'without-access'}
+                  />
+                  <i className='tier-access-selector__customized-radio-input' />
+                  <label htmlFor='without-access' className='tier-access-selector__label'>Data without Access</label>
+                </div>
+                <div className='tier-access-selector__item'>
+                  <input
+                    onChange={this.handleSelectorChange}
+                    name='access-level'
+                    id='all-data'
+                    value='all-data'
+                    className='tier-access-selector__radio-input'
+                    type='radio'
+                    checked={this.state.selected === 'all-data'}
+                  />
+                  <i className='tier-access-selector__customized-radio-input' />
+                  <label htmlFor='all-data' className='tier-access-selector__label'>All Data</label>
+                </div>
+              </div>
+              {
+                this.state.selected !== 'with-access' && (
+                  <div className='tier-access-selector__button-wrapper'>
+                    <Button
+                      label='Get Access'
+                      className='tier-access-selector__button'
+                      buttonType='default'
+                      onClick={
```

Comment:
can we just make the get access prop required? it seems weird to have a button that goes nowhere if the prop isn't passed in

Reply:
yeah sure could we have a default url for this button?

Reply:
I think we left this prop to be optional is because we want this button's behavior to be more customizable, like displaying a toaster by default?

But if we give it a default URL, what would the URL be?

Reply:
change default url to https://gen3.org/